tests/testthat/test-brierScore.R

```diff
@@ -0,0 +1,252 @@
+
+test_that("brierScore(SurvivalQuantities) returns same results as survreg", {
+    ### Setup Data
```

Comment:
maybe good to add here more explanation as comments why this should be the same (e.g. is it a special case of the Bayesian model etc.)